resources/settings.xml

```diff
@@ -5,9 +5,11 @@
     <setting label="32125" id="use_elementum_proxy" type="bool" default="true" />
     <setting id="elementum_proxy_help" label="32139" type="text" enable="false" subsetting="true" />
 
-    <setting label="32084" id="use_public_dns" type="bool" default="false" enable="eq(-2,false)" />
+    <setting label="32084" id="use_public_dns" type="bool" default="false" enable="eq(-2,false) + eq(3,false)" />
```

Comment:
@galeksandrp  are you sure about this +/- values? works properly when moving radio buttons?

Reply:
[Now](https://github.com/elgatito/script.elementum.burst/pull/297/commits/d368e090cc36616b5dee3d994257d621f1c621e9) I sure.

First `eq(` of `use_public_dns` child options shouldn't have been changed, because `use_public_dns` is not moved from their children.